.pep8.rc

```diff
@@ -0,0 +1,6 @@
+[pep8]
+max-line-length = 99
+show-source = enable
+statistics = enable
+count = enable
```

Comment:
As I have mentioned already on the bug in Bugzilla we don't want those three boolean keys included. Those are not necessary and seem to also cause the failure with Travis.